labgrid/driver/dediprogflashdriver.py

```diff
@@ -0,0 +1,77 @@
+import os.path
+import logging
+import attr
+
+from ..resource import NetworkDediprogFlasher
+from ..factory import target_factory
+from ..step import step
+from ..protocol import BootstrapProtocol
+from .common import Driver, check_file
+from ..util.managedfile import ManagedFile
+from ..util.helper import processwrapper
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class DediprogFlashDriver(Driver, BootstrapProtocol):
```

Comment:
@jluebbe @Emantor I always thought that the BootstrapProtocol was meant to write to non-persistent memory only (i.e. load an image into RAM). But the FlashromDriver seems to also implement it and now DediprogFlashDriver will, too?

In #206 we discussed that the QuartusHPSDriver explicitly does not implement the BootstrapProtocol. The plan was to introduce some kind of FlashProtocol instead later.

Reply:
Ok, will drop `BootstrapProtocol` on my next version. And don't really need `labgrid-client` support, as I'm using python library directly.